CoEpi/en.lproj/Localizable.strings

```diff
@@ -161,6 +119,10 @@
 "UX.home.alerts2" = "    Review your potential exposures";
 "UX.home.detected" = "        new exposures detected\n\n";
 "UX.home.how" = "How is my data being used?";
+"UX.home.share" = "Share";
+"UX.home.footer.build" = "Build";
+"UX.home.footer.debug" = "Debug";
+"UX.home.footer.version" = "Version";
```

Comment:
Small note: The "UX" prefix can be dropped. It doesn't have purpose. I missed this in the original PR. You can leave the existing translations like this, but don't use it for new ones.